fw/vhost.c

```diff
@@ -2733,56 +2801,66 @@ static TfwCfgSpec tfw_global_frang_specs[] = {
 	{
 		.name = "http_uri_len",
 		.deflt = "0",
-		.handler = tfw_cfgop_frang_uri_len,
+		.handler = tfw_cfgop_frang_glob_set_int,
+		.dest = &tfw_frang_vhost_reconfig.http_uri_len,
 		.allow_reconfig = true,
 	},
 	{
 		.name = "http_body_len",
 		.deflt = "1073741824", /* 1 Gb. */
-		.handler = tfw_cfgop_frang_body_len,
+		.handler = tfw_cfgop_frang_glob_set_long,
```

Comment:
It is incorrect to use `tfw_cfgop_frang_glob_set_long` here. If we have global frang limits like below:
frang_limits {
    http_body_len 0;
 }
 frang_limits {
 }

We should not override `http_body_len 0;` in the second config as i understand the idea in all other places

Reply:
I returned the old behavior, only explicit parameters should be overridden.